conda_rpms/generate.py

```diff
@@ -48,6 +49,14 @@ def render_dist_spec(dist, config):
                                 rpm_prefix=rpm_prefix,
                                 install_prefix=install_prefix)
 
+def render_env(branch_name, label, tag):
+    env_info = {'url': 'http://link/to/gh',
+                'name': branch_name,
+		'label' : label,
+                'summary': 'A SciTools environment.',
+                'version': '1',}
```

Comment:
Your worry is correct. The only trouble is how do we handle the versioning? How do we let it know that an `env-rpm` that points to a `taggedenv-rpm` already exists and that we now want a new version of that `env-rpm` that points at a different `taggedenv-rpm`?